apps/nextra/pages/en/build/smart-contracts/book/functions.mdx

```diff
@@ -232,9 +232,8 @@ script {
 }
 ```
 
-Entry functions can accept parameters that are: primitive types, reference to a `signer`, vectors (where the element type is itself acceptable),
-and certain standard library types such as `String`, `Object`, and `Option`.
-Entry functions must not have any return values.
+Entry functions can take primitive types, String, and vector arguments but cannot take Structs (e.g. Option). They also
```

Comment:
That's not true, see https://github.com/aptos-labs/aptos-core/blob/b3be2ceaf39f2542e5db41ed65b35b4eadcf8a6a/aptos-move/aptos-vm/src/verifier/transaction_arg_validation.rs#L54

Reply:
also nit: "can take .. as arguments", we also have type arguments and this reads a bit off

Reply:
OH damn it. That is were the zh diverges, I didn't edit any of this in fact, I just added this to zh first and then copied it over when I need to change to en/ :-/